best-current-practices/Selecting_an_SVID_format.md

```diff
@@ -0,0 +1,112 @@
+
+* [X.509-SVID](/standards/X509-SVID.md)
+* [JWT-SVID](/standards/JWT-SVID.md)
+* WIT-SVID (Workload Identity Token)
+
+This has been an additive process over the history of SPIFFE. The standard was originally written with X.509-SVID as the only supported SVID format, and JWT-SVID was subsequently added as the first token-based SVID to aid in interoperability with Layer 7 networking.
+
+More recently, WIT-SVID has been proposed as an additional token-based SVID format to make use of authentication concepts developed as part of the IETF [Workloads In Multi-System Environments (WIMSE)](https://datatracker.ietf.org/group/wimse/documents/) working group. This represents an enhancement over the existing JWT-SVID semantics by codifying proof-of-possession information as part of the mandatory claims of the token.
+
+## 2. Overview of Formats
+This section provides a brief description of the available SVID formats for contextual and comparative clarity. Full specification details for each can be found in their corresponding standardization (linked) documents.
+
+### 2.1 X.509-SVID
+[X.509-SVIDs](/standards/X509-SVID.md) are short-lived X.509 certificates containing a SPIFFE ID in the Subject Alternative Name. They are issued by the certificate authority (CA) of their SPIFFE trust domain, and used to authenticate during mutual TLS (mTLS) between workloads.
+
+Casting SVID information to an X.509 certificate format also allows for transport-level encryption to be used between peers with mutual TLS support, meaning X.509-SVIDs enable secure communication in transit between workloads that have such capabilities.
+
+### 2.2 JWT-SVID
+[JWT-SVIDs](/standards/JWT-SVID.md) are signed JSON Web Tokens (JWT) containing a SPIFFE ID in the `sub` claim. It is issued and signed by the JWT issuer of the SPIFFE trust domain, and is used to authenticate in the application layer between workloads.
+
+JWT-SVIDs are bearer tokens, and do not enable any kind of encryption or cryptographic protection against impersonation. Various recommendations exist to mitigate against known vulnerabilities of the JWT as an identity token, and should be followed when JWT-SVIDs are used in SPIFFE.
+
+### 2.3 WIT-SVID
+WIT-SVIDs are signed JWTs containing a SPIFFE ID in the `sub` claim and a public key binding, with proof-of-possession metadata regarding the signing key available in a `cnf` claim.
+
+WIT-SVIDs differ from JWT-SVIDs in that they are proof-of-possession tokens (i.e. **not** bearer tokens). This provides enhanced security guarantees about caller provenance, integrity, and non-repudiation while still maintaining the interoperability of using token-based authentication in the application layer.
+
+## 3. Comparison Matrix
+
+|                       | X.509-SVID                                     | JWT-SVID                                       | WIT-SVID                                                   |
```

Comment:
I think we should add the following rows:
* Integrity protection: X.509 yes, JWT no, WIT maybe
* Encryption: X.509 yes, JWT & WIT no
* Server authentication: X.509 yes, JWT & WIT no
* Support for TLS-termination between workloads: X.509 no, JWT & WIT yes